CHANGES/1485.removal

```diff
@@ -0,0 +1,2 @@
+Removed the optional "kid" parameter stored inside the signatures' payload generated during
+manifest v2 schema 1 conversion.
```

Comment:
generated during docker manifest v2 schema 1 conversion

Reply:
Okay. I am mentioning also the rationale behind the ecdsa removal.